boxnav/boxsim.py

```diff
@@ -56,20 +57,36 @@ def simulate(args: Namespace) -> None:
         action_taken, _ = agent.execute_navigator_action()
         print("Action taken:", action_taken)
 
-        actions_pbar.update()
-
         # Navigation progress is based on the percentage of the environment navigated
-        navigation_pbar.count = int(agent.get_percent_through_env())
-        navigation_pbar.update()
+        actions_pbar.update()
 
     if args.ue:
         agent.ue.close_osc()
 
-    navigation_pbar.close()
     actions_pbar.close()
 
     print("Simulation complete.")
 
+    if args.auto_upload:
+        chdir("..")
+        chdir("learning")
```

Comment:
Can this be done with one call to `chdir`?

Reply:
That is right, my mistake.